pages/_app.js

```diff
@@ -1,7 +1,16 @@
-import '../styles/globals.css'
+import Head from 'next/head';
+import '../styles/globals.css';
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <div>
+      <Head>
+        <title>instagram UI</title>
```

Comment:
It's no longer UI. You have a backend now. And the `i` should be capitalized.

Reply:
Copy that.
Renamed it to Instagram clone